examples/stand_alone/custom_main_pkgconfig/CMakeLists.txt

```
@@ -12,3 +12,5 @@ link_directories(${GAZEBO_LIBRARY_DIRS})
 
 add_executable(custom_main custom_main.cc)
 target_link_libraries(custom_main ${GAZEBO_LIBRARIES})
+
+target_link_directories(custom_main PUBLIC ${Boost_LIBRARY_DIRS})
```

this is causing an issue on bionic:

https://build.osrfoundation.org/job/gazebo-ci-pr_any-ubuntu_auto-amd64-gpu-nvidia/318/testReport/junit/(root)/Standalone_ExamplesBuild_Standalone/Standalone_5/

~~~
607: [ RUN      ] Standalone/ExamplesBuild_Standalone.Standalone/5
607: mkdir /tmp/custom_main_pkgconfig-d72c-9c6d-43a8-8e65
607: source /var/lib/jenkins/workspace/gazebo-ci-pr_any-ubuntu_auto-amd64-gpu-nvidia/gazebo/examples/stand_alone/custom_main_pkgconfig
607: -- The C compiler identification is GNU 5.4.0
607: -- The CXX compiler identification is GNU 5.4.0
607: -- Check for working C compiler: /usr/lib/ccache/cc
607: -- Check for working C compiler: /usr/lib/ccache/cc -- works
607: -- Detecting C compiler ABI info
607: -- Detecting C compiler ABI info - done
607: -- Detecting C compile features
607: -- Detecting C compile features - done
607: -- Check for working CXX compiler: /usr/lib/ccache/c++
607: -- Check for working CXX compiler: /usr/lib/ccache/c++ -- works
607: -- Detecting CXX compiler ABI info
607: -- Detecting CXX compiler ABI info - done
607: -- Detecting CXX compile features
607: -- Detecting CXX compile features - done
607: -- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.1") 
607: -- Checking for module 'gazebo'
607: --   Found gazebo, version 9.14.0
607: CMake Error at CMakeLists.txt:16 (target_link_directories):
607:   Unknown CMake command "target_link_directories".
~~~

---

should we just revert this line?

---

If it can not be removed, this might work:
```suggestion
if(${CMAKE_VERSION} VERSION_LESS "3.13.0") 
  link_directories(${Boost_LIBRARY_DIRS})
else()
  target_link_directories(custom_main PUBLIC ${Boost_LIBRARY_DIRS})
endif()
```